experiments/Markov/src/StochExpr.jl

```diff
@@ -0,0 +1,46 @@
+using Catlab
+using Catlab.Doctrines
+import Base: length
+
+export crand
+
+length(::FreeCartesianCategory.Ob{:generator}) = 1
```

Comment:
This convenient function is also ready implemented in `Catlab.Doctrines` as `ndims`. Maybe it should be `length`? Depends on how you think about the object.